pkg/model/test.go

```diff
@@ -34,8 +34,8 @@ func (test *Test) expandEnvVars() error {
 }
 
 func (t *Test) UnmarshalYAML(unmarshal func(interface{}) error) error {
-	type alias Test
-	var tt alias
+	type testAlias Test
```

Comment:
I renamed the two different `alias` types into `testAlias` and `testAliasCommand` so that we can differenciate between them, otherwise we always get `model.alias` in the unmarshalling error